api/v1alpha1/openapi.yaml

```diff
@@ -2064,6 +2064,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+        "500":
+          description: InternalServerError
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```

Comment:
Explicitly declaring 500 response codes shouldn't be necessary (they happen if we return a non-nil error, see below).

Reply:
Removed from schema.